ICAFIX/scripts/computeRecleanFeatures.m

```diff
@@ -0,0 +1,273 @@
+
+        CORTEX_end_idx=max(cifti_diminfo_dense_get_surface_info(CIFTI.diminfo{1}, 'CORTEX_RIGHT').ciftilist);
+        CEREBELLUM_start_idx=min(cifti_diminfo_dense_get_volume_structure_info(CIFTI.diminfo{1}, 'CEREBELLUM_LEFT').ciftilist);
+        CEREBELLUM_end_idx=max(cifti_diminfo_dense_get_volume_structure_info(CIFTI.diminfo{1}, 'CEREBELLUM_RIGHT').ciftilist);
+        BRIAN_STEM_start_idx=min(cifti_diminfo_dense_get_volume_structure_info(CIFTI.diminfo{1}, 'BRAIN_STEM').ciftilist);
+        BRIAN_STEM_end_idx=min(cifti_diminfo_dense_get_volume_structure_info(CIFTI.diminfo{1}, 'BRAIN_STEM').ciftilist);
+        
+        row_names=cell(num_comps, 1);
+        probs=zeros(num_comps,1);
+        for k=1:num_comps
+            row_names{k}=[subj '_' fMRIName '_' num2str(k)];
+            probs(k,:)=probability(k);
+            OrigFeatures(k,:)=FeaturesFile(k,:);
+
+            NewFeatures(k,1)=Stats(k,1); %Percent explained variance
+            NewFeatures(k,2)=Stats(k,2); %Percent total variance
+            NewFeatures(k,3)=sum(Powerspectra(PowerspectraScale>0.25,k))./sum(Powerspectra((PowerspectraScale>0.005).*(PowerspectraScale<0.1)==1,k)); %Ratio of power spectrum less than 0.005 to BOLD range
+            NewFeatures(k,4)=sum(Powerspectra(PowerspectraScale<0.005,k))./sum(Powerspectra((PowerspectraScale>0.005).*(PowerspectraScale<0.1)==1,k)); %Ratio of power spectrum greater than 0.25 to BOLD range
+            NewFeatures(k,5)=sum(abs(sICAweighted(sICAnoisy,k)))./sum(abs(sICAweighted(:,k))); %Ratio of amplitude at noisy (top 12.5% variance) timepoints to all timepoints weighted by explained variance
+            NewFeatures(k,6)=sum(abs(sICA(sICAnoisy,k)))./sum(abs(sICA(:,k))); %Ratio of amplitude at noisy (top 12.5% variance) timepoints to all timepoints unweighted
+            NewFeatures(k,7)=sum(abs(CIFTI.cdata(:,k)))./sum(abs(Volume_data(:,k))); %CIFTI/Volume
+            NewFeatures(k,8)=sum(abs(CIFTI.cdata(CORTEX_start_idx:CORTEX_end_idx,k)))./sum(abs(Volume_data(:,k))); %Cerebral Cortex/Volume
+            NewFeatures(k,9)=sum(abs(CIFTI.cdata(CORTEX_end_idx:end,k)))./sum(abs(Volume_data(:,k))); %Subcortical/Volume
+            NewFeatures(k,10)=sum(abs(CIFTI.cdata(CEREBELLUM_start_idx:CEREBELLUM_end_idx,k)))./sum(abs(Volume_data(:,k))); %Cerebellum/Volume
+            NewFeatures(k,11)=sum(abs(CIFTI.cdata(BRIAN_STEM_start_idx:BRIAN_STEM_end_idx,k)))./sum(abs(Volume_data(:,k))); %Brainstem/Volume
+            NewFeatures(k,12)=sum(abs(CIFTI.cdata(setdiff(1:CIFTI_LENGTH,[CORTEX_start_idx:CEREBELLUM_end_idx BRIAN_STEM_start_idx:BRIAN_STEM_end_idx CEREBELLUM_start_idx:CEREBELLUM_end_idx]),k)))./sum(abs(Volume_data(:,k))); %Diencephalon/Volume
+            NewFeatures(k,13)=sum(abs(CIFTI.cdata(CORTEX_start_idx:CORTEX_LEFT_end_idx,k)))./sum(abs(CIFTI.cdata(CORTEX_RIGHT_end_idx:CEREBELLUM_end_idx,k))); %Left Cerebral Cortex/Right Cerebral Cortex
```

Comment:
Additional CEREBELLUM typo, affecting LR symmetry feature.

Reply:
Also, CORTEX_RIGHT_end_idx is the start index?  Using the lists instead of ranges probably would have avoided these errors.